docs/content/how-to/disconnected/known-issues.md

```diff
@@ -44,3 +45,100 @@ data:
+{"level":"error","ts":"2024-02-14T22:18:12Z","msg":"Reconciler error","controller":"hostedcluster","controllerGroup":"hypershift.openshift.io","controllerKind":"HostedCluster","HostedCluster":{"name":"simone3","namespace":"local-cluster"},"namespace":"local-cluster","name":"simone3","reconcileID":"6d6a2479-3d54-42e3-9204-8d0ab1013745","error":"failed to look up image metadata for registry.ci.openshift.org/ocp/4.14-2024-02-14-135111@sha256:84c74cc05250d0e51fe115274cc67ffcf0a4ac86c831b7fea97e484e646072a6: failed to obtain root manifest for registry.ci.openshift.org/ocp/4.14-2024-02-14-135111@sha256:84c74cc05250d0e51fe115274cc67ffcf0a4ac86c831b7fea97e484e646072a6: unauthorized: authentication required","stacktrace":"sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler\n\t/remote-source/app/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:326\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem\n\t/remote-source/app/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:273\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2\n\t/remote-source/app/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:234"}
+```
+
+**Workaround:**
+Explictly use --annotations=hypershift.openshift.io/control-plane-operator-image=<controlPlaneOperatorImage> when creatng the hosted cluster.
+```bash
+$ PAYLOADIMAGE=$(oc get clusterversion version -ojsonpath='{.status.desired.image}')
+$ HO_OPERATOR_IMAGE="${PAYLOADIMAGE//@sha256:[^ ]*/@\$(oc adm release info -a /tmp/.dockerconfigjson "$PAYLOADIMAGE" | grep hypershift | awk '{print $2}')}"
+$ hcp create cluster ... --annotations=hypershift.openshift.io/control-plane-operator-image=${HO_OPERATOR_IMAGE} ...
+```
+
+## (KubeVirt specific) oc-mirror does not mirror the RHCOS image for HyperShift KubeVirt provider from the OCP release payload
+
+See: [OCPBUGS-29408](https://issues.redhat.com/browse/OCPBUGS-29408)
+
+**Workaround:**
+Explicitly mirror the RHCOS image for HyperShift KubeVirt provider.
+```bash
+mirror_registry=$(oc get imagecontentsourcepolicy -o json | jq -r '.items[].spec.repositoryDigestMirrors[0].mirrors[0]')
+mirror_registry=${mirror_registry%%/*}
+LOCALIMAGES=localimages
+PAYLOADIMAGE=$(oc get clusterversion version -ojsonpath='{.status.desired.image}')
+mkdir -p /tmp/release-manifests/
+oc image extract ${PAYLOADIMAGE} --path /release-manifests/:/home/release-manifests/ --confirm
+RHCOS_IMAGE=$(cat /home/release-manifests/0000_50_installer_coreos-bootimages.yaml | yq -r .data.stream | jq -r '.architectures.x86_64.images.kubevirt."digest-ref"')
+RHCOS_IMAGE_NO_DIGEST=${RHCOS_IMAGE%@sha256*}
+RHCOS_IMAGE_NAME=${RHCOS_IMAGE_NO_DIGEST##*/}
+RHCOS_IMAGE_REPO=${RHCOS_IMAGE_NO_DIGEST%/*}
+oc image mirror ${RHCOS_IMAGE} ${mirror_registry}/${LOCALIMAGES}/${RHCOS_IMAGE_NAME}
```

Comment:
This looks good to me as  a workaround for the KubeVirt rhcos image (for the time being until oc-mirror does what we need)